tests/unit/type/hash.tcl

```diff
@@ -350,7 +350,7 @@ start_server {tags {"hash"}} {
         set _ $rv
     } {{{} {}} {{} {}} {{} {}}}
 
-    test {HMGET HRANDFIELD HGET HGETALL HDEL HINCRBY HINCRBYFLOAT HSTRLEN against wrong type} {
+    test {HKEYS HVALS HEXISTS HMGET HRANDFIELD HGET HGETALL HDEL HINCRBY HINCRBYFLOAT HSTRLEN against wrong type} {
```

Comment:
let's call it `Hash commands against wrong type` instead of modifying the test name again and again.